### PR TITLE
fix(fb pixel): ecomm

### PR DESCRIPTION
**Pull request by Gauravudia**
## Description of the change

> Fixing Ecomm

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/destinations/facebook_pixel/transform.js [hunk 1]
```diff
@@ -200,6 +204,42 @@ const handleProduct = (message, categoryToContent, valueFieldIdentifier) => {
   };
 };
 
+const handleSearch = (message) => {
```
Comment (shrouti1507) [inline src/v0/destinations/facebook_pixel/transform.js:207]: can not find any use of `valueFieldIdentifier` but it is used while calling the function

Comment (Gauravudia) [reply src/v0/destinations/facebook_pixel/transform.js:207]: updated.

**Review by shrouti1507 — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Review by shrouti1507 — DISMISSED** [review]

**Review by shrouti1507 — APPROVED** [review]

File: src/v0/destinations/facebook_pixel/transform.js [hunk 2]
```diff
@@ -200,6 +204,42 @@ const handleProduct = (message, categoryToContent, valueFieldIdentifier) => {
   };
 };
```
Comment (krishna2020) [inline src/v0/destinations/facebook_pixel/transform.js:206]: Write unit test for `handleSearch` function under `/test/integrations/destinations/facebook_pixel/transform.js`
cc: @saikumarrs

Comment (krishna2020) [reply src/v0/destinations/facebook_pixel/transform.js:206]: Created this PR - https://github.com/rudderlabs/rudder-transformer/pull/1964/files
Can you add more test cases to it with different scenarios ? You can use the same payload & modify the parameter values before executing a test case.

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**